packages/amplify-cli-core/src/state-manager/pathManager.ts

```diff
@@ -35,6 +39,7 @@ export const PathConstants = {
   LocalAWSInfoFileName: 'local-aws-info.json',
   TeamProviderInfoFileName: 'team-provider-info.json',
   BackendConfigFileName: 'backend-config.json',
+  OverrideFileName: 'override.js',
```

Comment:
I thought we were supporting CDK based overrides, and making use of typsecript. Has anything changed to use `.js` file